UI/GamepadEmu.cpp

```diff
@@ -215,8 +214,9 @@ void AnalogRotationButton::Update() {
 	if (autoRotating_) {
 		float speed = clockWise_ ? -g_Config.fAnalogAutoRotSpeed : g_Config.fAnalogAutoRotSpeed;
 		// Clamp to a square
-		__CtrlSetAnalogX(std::min(1.0f, std::max(-1.0f, 1.42f*cosf(now*speed))), 0);
-		__CtrlSetAnalogY(std::min(1.0f, std::max(-1.0f, 1.42f*sinf(now*speed))), 0);
+		__CtrlSetAnalogXY(0,
+			std::min(1.0f, std::max(-1.0f, 1.42f*cosf(now*speed))),
```

Comment:
IMHO, a lot of tools (whether it be grep or other file search, compiler error messages, etc.) show things by line.

I personally think whenever you start hard wrapping statements, it's a sign that the code is actually just bad.  Wrapping the code to multiple lines is a bandage to already wounded code.  Temporaries are better, and result in code that is more readable, errors and search results that are more useful, etc.

Of course assignments and etc. are different, but I prefer avoided wrapping function calls, using named temporaries or structs.

-[Unknown]

Reply:
Yeah, that's definitely a good point. I'm not quite as averse to them in general, but here temporaries would clearly be better indeed.